.dev/config/webpack.config.js

```diff
@@ -13,10 +13,13 @@ module.exports = {
 	...defaultConfig,
 
 	entry: {
-		'coblocks': path.resolve( process.cwd(), 'src/blocks.js' ),
+		coblocks: path.resolve( process.cwd(), 'src/blocks.js' ),
+
+		// Styles
 		'coblocks-editor': path.resolve( process.cwd(), 'src/styles/editor.scss' ),
 		'coblocks-style': path.resolve( process.cwd(), 'src/styles/style.scss' ),
 
+		// Front-End Scripts
```

Comment:
Most of those should also be removed but let's leave that for a follow-up PR. cc @fullofcaffeine

Reply:
https://github.com/Automattic/coblocks/pull/8